FBSnapshotTestCase/FBSnapshotTestCasePlatform.m

```diff
@@ -34,7 +34,7 @@ BOOL FBSnapshotTestCaseIs64Bit(void)
 NSString *FBDeviceAgnosticNormalizedFileName(NSString *fileName)
 {
   UIDevice *device = [UIDevice currentDevice];
-  CGSize screenSize = [[UIApplication sharedApplication] keyWindow].bounds.size;
```

Comment:
Actually, just had a realization: the problem doesn't have anything to do with frameworks.  Our app [short-circuits itself when being run as the test host](https://github.com/wikimedia/wikipedia-ios/blob/master/Wikipedia/Code/main.m#L10), which means no `AppDelegate`, which means no `keyWindow`.

So, this is more about handling our specific edge case, which means that a window should be made the `keyWindow` before suffixes are generated.  At which point, I'm wondering if we should be using a "dummy" app delegate (which just has a dummy window), or FBSnapshotTestCase should install one for me (e.g. if you were running a Logic Tests target which didn't even have a host).